### PR TITLE
chore: update SDK

### PR DESCRIPTION
**Pull request by shuplenkov**
## Issue being fixed or feature implemented

Need to update SDK & wallet-lib to the latest versions

## What was done?

Updated SDK & wallet-lib dependencies

## How Has This Been Tested?

With tests

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,12 +49,12 @@
...
     "@dashevo/dashcore-lib": "~0.19.25",
-    "@dashevo/dashd-rpc": "~2.2.2",
+    "@dashevo/dashd-rpc": "~2.3.0",
     "@dashevo/dashpay-contract": "github:dashevo/dashpay-contract#update-dpp",
     "@dashevo/dpns-contract": "github:dashevo/dpns-contract#update-dpp",
-    "@dashevo/dpp": "~0.20.0-dev.2",
+    "@dashevo/dpp": "~0.20.0-dev.3",
     "@dashevo/feature-flags-contract": "github:dashevo/feature-flags-contract#update-dpp",
-    "@dashevo/wallet-lib": "~7.20.0-dev.3",
+    "@dashevo/wallet-lib": "github:dashevo/wallet-lib#update-dapi-client",
```
Comment (shumkov) [inline package.json:57]: Use dev version.

File: package.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "awilix": "^4.2.6",
     "bls-signatures": "^0.2.5",
     "chalk": "^4.0.0",
-    "dash": "~3.20.0-dev.3",
+    "dash": "github:dashevo/js-dash-sdk#feat/more-proofs",
```
Comment (shumkov) [inline package.json:66]: Use dev version.

File: src/listr/tasks/platform/initTaskFactory.js [hunk 3]
```diff
@@ -51,17 +51,21 @@ function initTaskFactory(
             clientOpts.dapiAddresses = [ctx.dapiAddress];
           }
 
+          const nodeCount = ctx.nodeCount || 1;
```
Comment (shumkov) [inline src/listr/tasks/platform/initTaskFactory.js:54]: Does it help or not?

Comment (shuplenkov) [reply src/listr/tasks/platform/initTaskFactory.js:54]: No, I'll remove it

**Review by jawid-h — APPROVED** [review]

**Review by jawid-h — APPROVED** [review]

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**